### PR TITLE
Add NXP T2080 PPC

### PR DESCRIPTION
**Pull request by elms**

### CODE REVIEW

File: hal/t2080.c [hunk 1]
```diff
@@ -0,0 +1,151 @@
+#include <stdint.h>
```
Comment (dgarske) [inline hal/t2080.c:1]: Please add our standard GPL header.

File: hal/t2080.c [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+#ifdef DEBUG_UART
+#define UART0_OFFSET 0x11C500
+#define UART0_BASE  (CCSRBAR + UART0_OFFSET)
+
+#define SYS_CLK 600000000
+#define BAUD_RATE 115200
+
+static inline uint8_t in_8(const volatile unsigned char *addr)
+{
+	uint8_t ret;
```
Comment (dgarske) [inline hal/t2080.c:16]: Can we use 4 spaces instead of tab?

File: src/boot_ppc.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#include <stdint.h>
```
Comment (dgarske) [inline src/boot_ppc.c:1]: Add our standard GPL header.

File: src/boot_ppc_start.S [hunk 4]
```diff
@@ -0,0 +1,357 @@
+/*
```
Comment (dgarske) [inline src/boot_ppc_start.S:1]: Add standard GPL header.

File: test-app/app_t2080.c [hunk 5]
```diff
@@ -0,0 +1,99 @@
+#include <stdint.h>
```
Comment (dgarske) [inline test-app/app_t2080.c:1]: GPL header and 4 spaces not tabs.

File: test-app/app_t2080.c [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    while (sz-- > 0) {
+        while (!(in_8(uart + 5) & 0x20))
+		;
+        out_8(uart + 0, buf[pos++]);
+    }
+}
+
+#endif /* DEBUG_UART */
+
+static char* hex_lut = "0123456789abcdef";
```
Comment (dgarske) [inline test-app/app_t2080.c:73]: Can this be `const`? Should be wrapped in `#ifdef DEBUG_UART`.

**Review by dgarske — CHANGES_REQUESTED** [review]

File: test-app/app_t2080.c (hunk 5 above)
Comment (elms) [reply test-app/app_t2080.c:22]: Done

File: src/boot_ppc_start.S (hunk 4 above)
Comment (elms) [reply src/boot_ppc_start.S:23]: Done

File: src/boot_ppc.c (hunk 3 above)
Comment (elms) [reply src/boot_ppc.c:22]: Done

File: hal/t2080.c (hunk 1 above)
Comment (elms) [reply hal/t2080.c:21]: Done

File: hal/t2080.c (hunk 2 above)
Comment (elms) [reply hal/t2080.c]: Done

File: test-app/app_t2080.c (hunk 6 above)
Comment (elms) [reply test-app/app_t2080.c]: Done

**Review by dgarske — APPROVED** [review]

**Comment by dgarske** [comment]
Over to @danielinux for final review.

**Review by danielinux — APPROVED** [review]

**Pull request merged**